code/datums/components/chasm.dm

```diff
@@ -85,10 +85,15 @@
 			var/mob/living/carbon/human/H = AM
 			if(istype(H.belt, /obj/item/wormhole_jaunter))
 				var/obj/item/wormhole_jaunter/J = H.belt
-				//To freak out any bystanders
-				H.visible_message(span_boldwarning("[H] falls into [parent]!"))
-				J.chasm_react(H)
-				return FALSE
+				var/list/D = J.get_destinations()
```

Comment:
please don't call variables D. If you wanna be really cool fix up the rest of this proc, but at the least don't make things worse https://github.com/tgstation/tgstation/blob/master/.github/CONTRIBUTING.md#use-descriptive-and-obvious-names

Reply:
Fixed with https://github.com/tgstation/tgstation/pull/61055/commits/dac49ff868b4271b610b011dcacd65a5aa167526